### PR TITLE
Update Unraid docs with steps to generate certificates 

### PR DESCRIPTION
**Pull request by paulhovey**
Updates and cleans up a few other things too.  I don't personally use CA, so added a few details for those who wish to setup without using it.

### CODE REVIEW

File: docs/Installing/Unraid.md [hunk 1]
```diff
@@ -4,26 +4,30 @@ title: 'Unraid'
...
 select the one by Kippenhof (the only one at this time).
 1. Click Install.
 
    ![image](https://user-images.githubusercontent.com/2792750/180338271-ca70f0d3-8f23-4d00-9cdb-ea011975dab3.png)
 
+
+### Using Docker without Community Apps
+
+1.  Create a new image and follow the guidelines below to configure the parameters
+This image is not very complex, and can easily be configured in the web interface.
```
Comment (j-f1) [inline docs/Installing/Unraid.md:24]: I wouldn’t want someone to see this and get discouraged if it isn’t easy for them. Would you be willing to give a more complete step-by-step description?

File: docs/Installing/Unraid.md [hunk 2]
```diff
@@ -51,12 +56,36 @@ instructions.
...
    ![image](https://user-images.githubusercontent.com/2792750/180340859-814bac85-090e-4ac6-a814-56f13c017845.png)
 1. Confirm that the correct image is pulled. The `By:` section should read `By:
-jlongster/actual-server` (or whichever image source you have chosen to use).
+ghcr.io/actualbudget/actual-server:latest` (or whichever image source you have chosen to use).
 ![image](https://user-images.githubusercontent.com/2792750/180320492-0f5977e0-15e4-4640-9d07-db66806a33a2.png)
 
+## Advanced
+### Creating HTTPS Certificates
+
+New versions of Actual depend on HTTPS to be enabled (see [Enabling SharedArrayBuffer Access](https://actualbudget.github.io/docs/Troubleshooting/SharedArrayBuffer)).  To create your own certificate on Unraid, open up a terminal into your server to follow these steps:
```
Comment (j-f1) [inline docs/Installing/Unraid.md:68]:
```suggestion
New versions of Actual depend on HTTPS to be enabled (see [Enabling SharedArrayBuffer Access](/Troubleshooting/SharedArrayBuffer)).  To create your own certificate on Unraid, open up a terminal into your server to follow these steps:
```

File: docs/Installing/Unraid.md (hunk 1 above)
Comment (paulhovey) [reply docs/Installing/Unraid.md:24]: ha I put that in there trying to encourage that it was easy :) added another sentence describing the necessary parameters to configure.  They're all documented below, so didn't want to duplicate more than necessary.

Comment (j-f1) [reply docs/Installing/Unraid.md:24]: Ok, that makes sense! Thanks for clarifying :)

File: docs/Installing/Unraid.md [hunk 3]
```diff
@@ -51,12 +57,36 @@ instructions.
...
+## Advanced
+### Creating HTTPS Certificates
+
+New versions of Actual depend on HTTPS to be enabled (see [Enabling SharedArrayBuffer Access](/Troubleshooting/SharedArrayBuffer)).  To create your own certificate on Unraid, open up a terminal into your server to follow these steps:
+
+1.  Change directory into your shared data folder (see "Setting up data persistence" above)
+1.  Create a new folder, "keys"
+1.  Change directory into the "keys" folder
+1.  Generate a certificate and private key combo by running `openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout actual.key -out actual.crt`
+1.  In the Docker settings on the WebUI, create a new Variable "ACTUAL_HTTPS_KEY" and set the value to the path of the newly created key, `/data/keys/actual.key` (note: this is the value of the path _inside_ the docker container)
```
Comment (shall0pass) [inline docs/Installing/Unraid.md:75]: "ACTUAL_HTTPS_KEY" doesn't seem to be rendering properly in the preview.

Comment (j-f1) [reply docs/Installing/Unraid.md:75]:
```suggestion
1.  In the Docker settings on the WebUI, create a new Variable `ACTUAL_HTTPS_KEY` and set the value to the path of the newly created key, `/data/keys/actual.key` (note: this is the value of the path _inside_ the docker container)
```

Comment (paulhovey) [reply docs/Installing/Unraid.md]: Changed the quotation style to your suggestion :+1:

**Comment by shall0pass** [comment]
LGTM

**Review by j-f1 — APPROVED** [review]

**Pull request merged**